app/app.go

```diff
@@ -294,6 +297,7 @@ func New(
 		authzkeeper.StoreKey,
 		evmtypes.StoreKey,
 		feemarkettypes.StoreKey,
+		erc20types.StoreKey,
```

Comment:
I would assume yes, and it should be tested with the upgrade tests, but the storage keys are all similar as Ethermint if we want to keep the data?